docs/overview/01-introduction_what_is_zkverify/03-core_components.md

```diff
@@ -41,3 +41,53 @@ We have created built-in verifier pallets for different proving schemes to suppo
 - Risc Zero
 - SxT Proof of SQL
 
+## Domains
```

Comment:
This info seems to be too much detailed for this section here (which is supposed to be a general overview of the project). Please considering moving it somewhere else